misc/scripts/install_vulkan_sdk_macos.sh

```diff
@@ -3,6 +3,11 @@
 set -euo pipefail
 IFS=$'\n\t'
 
+if [ -d "$HOME/VulkanSDK" ]; then
```

Comment:
If I recall correctly godot engine uses the builtin system. Might want to investigate reusing that. This is simpler though.